src/componentTest/resources/expectedTemplateContent/service-bus-consumer/src/main/resources/application-vstsTest.properties

```diff
@@ -1 +1 @@
-servicebus.consumer.consumer-connection-url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Fnamespace.servicebus.windows.net%2Fconsumer%2Fsubscriptions%2Fsubscription_name&sig=token&se=1696020226&skn=shared_access_profile_listen // TODO: Create REX service bus topic and replace
+servicebus.consumer.consumer-connection-url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Ftest.servicebus.windows.net%2Fconsumer%2Fsubscriptions%2Ftest&sig=test&se=2147483646
```

Comment:
You lost a note here: `// TODO: Create REX service bus topic and replace`

Reply:
this is important to ensure ppl use valid connection strings for their integration tests locally.